pychord/constants/qualities.py

```diff
@@ -54,6 +55,7 @@
     ('add11', (0, 4, 7, 17)),
     ('4', (0, 4, 7, 17)),
     # 5 notes
+    ('m7add4', (0, 3, 5, 7, 22)),  # https://www.worshipchords.net/piano-chords/C/m7add4
```

Comment:
I think standard m7add4 is `(0, 3, 5, 7, 10)`.
https://www.solfej.io/chords/c-m7add4

Reply:
i agree
then maybe the method names should be refined (https://arlobelshee.com/good-naming-is-a-process-not-a-single-step/) or with an extra param to clearly provide "voicing" capabilities and handle https://github.com/yuma-m/pychord/issues/85 and https://github.com/yuma-m/pychord/pull/84/files#r1083399429.
what do you think?